sonar-db/src/main/gen-java/org/sonar/db/protobuf/DbCommons.java

```diff
@@ -0,0 +1,818 @@
+      }
+    }
+    public static final com.google.protobuf.Descriptors.Descriptor
+        getDescriptor() {
+      return org.sonar.db.protobuf.DbCommons.internal_static_sonarqube_db_commons_TextRange_descriptor;
+    }
+
+    protected com.google.protobuf.GeneratedMessage.FieldAccessorTable
+        internalGetFieldAccessorTable() {
+      return org.sonar.db.protobuf.DbCommons.internal_static_sonarqube_db_commons_TextRange_fieldAccessorTable
+          .ensureFieldAccessorsInitialized(
+              org.sonar.db.protobuf.DbCommons.TextRange.class, org.sonar.db.protobuf.DbCommons.TextRange.Builder.class);
+    }
+
+    public static com.google.protobuf.Parser<TextRange> PARSER =
+        new com.google.protobuf.AbstractParser<TextRange>() {
+      public TextRange parsePartialFrom(
+          com.google.protobuf.CodedInputStream input,
+          com.google.protobuf.ExtensionRegistryLite extensionRegistry)
+          throws com.google.protobuf.InvalidProtocolBufferException {
+        return new TextRange(input, extensionRegistry);
+      }
+    };
+
+    @java.lang.Override
+    public com.google.protobuf.Parser<TextRange> getParserForType() {
+      return PARSER;
+    }
+
+    private int bitField0_;
+    public static final int START_LINE_FIELD_NUMBER = 1;
+    private int startLine_;
+    /**
+     * <code>optional int32 start_line = 1;</code>
+     *
+     * <pre>
+     * Start line. Should never be absent
+     * </pre>
+     */
+    public boolean hasStartLine() {
+      return ((bitField0_ & 0x00000001) == 0x00000001);
+    }
+    /**
+     * <code>optional int32 start_line = 1;</code>
+     *
+     * <pre>
+     * Start line. Should never be absent
+     * </pre>
+     */
+    public int getStartLine() {
+      return startLine_;
+    }
+
+    public static final int END_LINE_FIELD_NUMBER = 2;
+    private int endLine_;
+    /**
+     * <code>optional int32 end_line = 2;</code>
+     *
+     * <pre>
+     * End line (inclusive). Absent means it is same as start line
+     * </pre>
+     */
+    public boolean hasEndLine() {
+      return ((bitField0_ & 0x00000002) == 0x00000002);
+    }
+    /**
+     * <code>optional int32 end_line = 2;</code>
+     *
+     * <pre>
+     * End line (inclusive). Absent means it is same as start line
+     * </pre>
+     */
+    public int getEndLine() {
+      return endLine_;
+    }
+
+    public static final int START_OFFSET_FIELD_NUMBER = 3;
+    private int startOffset_;
+    /**
+     * <code>optional int32 start_offset = 3;</code>
+     *
+     * <pre>
+     * If absent it means range starts at the first offset of start line
+     * </pre>
+     */
+    public boolean hasStartOffset() {
+      return ((bitField0_ & 0x00000004) == 0x00000004);
+    }
+    /**
+     * <code>optional int32 start_offset = 3;</code>
+     *
+     * <pre>
+     * If absent it means range starts at the first offset of start line
+     * </pre>
+     */
+    public int getStartOffset() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1161)